components/Header.tsx

```diff
@@ -0,0 +1,52 @@
+import { H, Level } from 'react-accessible-headings';
+import { Domine } from '@next/font/google'
+
+const domine = Domine({
+    weight: '500',
+    subsets: ['latin'],
+})
+
+function Header() {
+    return (
+        <div className="bg-[url('/images/bio_bg.jpg')]">
```

Comment:
it's better to use html img than css background, e.g. because of accessibility and optimization

Reply:
i will change that ASAP